cg/server/api.py

```diff
@@ -212,16 +212,16 @@ def parse_samples():
     if request.args.get("status") and not g.current_user.is_admin:
         return abort(http.HTTPStatus.FORBIDDEN)
     if request.args.get("status") == "incoming":
-        samples_q: List[Sample] = db.get_samples_to_receive()
+        samples: List[Sample] = db.get_samples_to_receive()
     elif request.args.get("status") == "labprep":
-        samples_q: List[Sample] = db.get_samples_to_prepare()
+        samples: List[Sample] = db.get_samples_to_prepare()
     elif request.args.get("status") == "sequencing":
-        samples_q: List[Sample] = db.get_samples_to_sequence()
+        samples: List[Sample] = db.get_samples_to_sequence()
     else:
         customer_objs: Optional[Customer] = (
```

Comment:
The type hint is incorrect. If we want to use type hints here it should be Optional[List[Customer]]